content/environment_variables.tex

```diff
@@ -30,14 +30,30 @@
     \tabularnewline\hline
 %%
 \EnvVarDecl{SHMEM\_SYMMETRIC\_SIZE}
-    & Non-negative integer
-    & Number of bytes to allocate for symmetric heap
+    & Positive decimal value (integer or floating point) with an optional
+    character suffix %that specifies a scaling factor
+    & The value set in this environment variable is interpreted as an
+    implementation-defined size (in bytes) at least as large as the integer
+    ceiling of the product of the numeric prefix and the scaling factor. The
+    allowed character suffix for the scaling factor are as follows:
+      \begin{itemize}
+        \item k or K multiplies by \(2^{10}\)  (kilobytes)
+        \item m or M multiplies by \(2^{20}\)  (megabytes)
+        \item g or G multiplies by \(2^{30}\)  (gigabytes)
+        \item t or T multiplies by \(2^{40}\)  (terabytes)
+      \end{itemize}
+      For example, the string 20m is equivalent to the integer value 20971520,
```

Comment:
from the review comments during Sep spec meeting: quotes for the values and add example for 0.5 vs .5 for the usage